--- libipcalc-1.0.1.orig/Makefile.in.orig	2011-01-11 15:00:40.000000000 +0900
+++ libipcalc-1.0.1/Makefile.in	2011-01-11 16:02:53.695398904 +0900
@@ -64,8 +64,8 @@ MANLANG         = ko
 
 all: $(OBJECT) $(UTIL_OBJ)
 	$(MAKE) libs
-	$(PLINK) -o ip2long ip2long.lo lib$(PROG)$(COMPAT).la
-	$(PLINK) -o netcalc netcalc.lo lib$(PROG)$(COMPAT).la
+	$(CC_OPT) -o ip2long ip2long.o -L.libs -l$(PROG)$(COMPAT)
+	$(CC_OPT) -o netcalc netcalc.o -L.libs -l$(PROG)$(COMPAT)
 
 ipcalc.lo: ipcalc.c ipcalc.h
 	$(COMPILE) -o $@ -c $<
