diff -urNp libipcalc.org//Makefile.in libipcalc/Makefile.in
--- libipcalc.org//Makefile.in	2011-01-11 20:01:59.265399004 +0900
+++ libipcalc/Makefile.in	2011-01-12 21:06:25.465398990 +0900
@@ -83,6 +83,11 @@ libs:
 install: install-header install-man
 	$(MKDIR) $(DESTDIR)$(bindir)
 	$(MKDIR) $(DESTDIR)$(libdir)/pkgconfig
+
+	for i in .libs/lib$(PROG)$(COMPAT).so.$(SONAME_VERSION) .libs/ip2long .libs/netcalc; do \
+		/usr/bin/chrpath -d $$i; \
+	done
+
 	$(P_LIBTOOL) $(PROG)-config $(DESTDIR)$(bindir)/$(PROG)$(COMPAT)-config
 	$(P_LIBTOOL) ip2long $(DESTDIR)$(bindir)/ip2long$(COMPAT)
 	$(P_LIBTOOL) netcalc $(DESTDIR)$(bindir)/netcalc$(COMPAT)
